app/main.py

```diff
@@ -39,10 +45,13 @@ def process(self, encrypted_jwt, tx_id=None):
 
         try:
             decrypted_payload = self._decrypt(encrypted_jwt, tx_id)
+            logger.debug("Decrypted message", tx_id=tx_id)
```

Comment:
The tx_id should be bound to the logger where possible to reduce chance of forgetting to log it